substra/cli/printers.py

```diff
@@ -302,6 +317,37 @@ class TraintuplePrinter(AssetPrinter):
     has_description = False
 
 
+class CompositeTraintuplePrinter(AssetPrinter):
+    asset_name = 'compositetraintuple'
+
+    list_fields = (
+        Field('Algo name', 'algo.name'),
+        Field('Status', 'status'),
+        Field('Perf', 'dataset.perf'),
+        Field('Tag', 'tag'),
+        Field('Compute Plan Id', 'computePlanID'),
+    )
+    single_fields = (
+        Field('Model key', 'outModel.hash'),
+        Field('Algo key', 'algo.hash'),
+        Field('Algo name', 'algo.name'),
+        Field('Objective key', 'objective.hash'),
+        Field('Status', 'status'),
+        Field('Perf', 'dataset.perf'),
+        Field('Dataset key', 'dataset.openerHash'),
+        KeysField('Train data sample keys', 'dataset.keys'),
+        InModelTraintupleKeysField('In model traintuple keys', 'inModels'),
```

Comment:
This was correct for a traintuple, but this is not the case for the composite traintuple.